### ISSUE TITLE
Dynamic pathmap improvement

### ISSUE THREAD

**Issue by ysroh** [ZeligsoftDev/CX4CBDDS#152]
# **Issue and tracking information**
Add "Remap" button to re-scan workspaces for dynamic pathmap.
Come up with a solution to handle two identical models in the workspace.

### Developer's Actual time spent on fix (hours)  

# **Issue reporter to provide a detailed description of the issue in the space below**

**Comment by ysroh** [comment]
I have tried to handle two models with same name and same pathmap but didn't work well. I don't think I can solve this issue deterministically as all mappings are recalculated upon restart. 
So two models with same name and same dynamic pathmap may cause problems to the mappings.
I suggest that you remove one of the model from the workspace either by deleting it or closing the project that contains the model. 

I added a "remap" button to recalculate the dynamic mappings so users don't need to restart the workspace to fix any issues with the dynamic mappings.

I also removed the "Original mapping" column as I don't think it adds any values to your use case and make users confuse more.
